src/util/cmdlineargs.h

```diff
@@ -44,6 +44,13 @@ class CmdlineArgs final {
     const QString& getResourcePath() const { return m_resourcePath; }
     const QString& getTimelinePath() const { return m_timelinePath; }
 
+    void storeScaleFactor(double scaleFactor) {
```

Comment:
Why "store"? This is just a simple setter. There is no side-effect.

Reply:
renamed to storeScaleFactor()